src/Evolution/Systems/Cce/AnalyticSolutions/RobinsonTrautman.cpp

```diff
@@ -48,30 +78,46 @@ void RobinsonTrautman::initialize_stepper_from_start() noexcept {
             l_max_, coefficient.l, -static_cast<int>(coefficient.m), 0)]);
   }
 
-  // re-use the `dense_output_rt_scalar_` buffers to reduce allocations
-  Spectral::Swsh::inverse_swsh_transform(
-      l_max_, 1, make_not_null(&get(dense_output_rt_scalar_)), libsharp_modes);
-  // offset the scalar to 'reasonable' value
-  get(dense_output_rt_scalar_).data() =
-      1.0 + std::complex<double>(1.0, 0.0) *
-                real(get(dense_output_rt_scalar_).data());
-  du_rt_scalar(make_not_null(&get(dense_output_du_rt_scalar_)),
-               get(dense_output_rt_scalar_));
-  stepper_ = boost::numeric::odeint::make_dense_output(
-      tolerance_ * 0.01, tolerance_,
-      boost::numeric::odeint::runge_kutta_dopri5<ComplexDataVector>{});
-  const double rt_scalar_scale = max(abs(get(dense_output_rt_scalar_).data()));
-  const double du_rt_scalar_scale =
-      max(abs(get(dense_output_du_rt_scalar_).data()));
-
-  double initial_step = extraction_radius_ * 1.0e-4 / square(l_max_);
-  if (rt_scalar_scale > 1.0e-5 and du_rt_scalar_scale > 1.0e-5) {
-    initial_step =
-        0.1 * rt_scalar_scale / (du_rt_scalar_scale * square(l_max_));
+  {
+    // re-use the `dense_output_rt_scalar_` buffers to reduce allocations
+    auto& initial_rt_scalar = dense_output_rt_scalar_;
+    auto& initial_du_rt_scalar = dense_output_du_rt_scalar_;
```

Comment:
I think these two variables could reference the `ComplexDataVector` directly, because below you `get` it every time anyway